README.md

```diff
@@ -34,17 +34,23 @@ If using React, you'll probably want to use `egghead-ui` as well; it is a librar
 
 ## Workflow
 
-- create a feature branch off of `master`
 - `yarn` to install latest packages
 - `yarn dev` to compile with a watcher
 - `yarn verify` check builds work
 - use [`yarn link`](https://yarnpkg.com/lang/en/docs/cli/link/) to test using changes in another project
-- if you want to publish a new library release, run `yarn bump` and type in the new version using [Semantic Versioning](http://semver.org/)
+    
+## Deployment
+
+After you have create a `feature-branch` off of `master`:
+- create a `feature-branch` off of `master`
```

Comment:
Don't need this line because right above it says "After you create a feature branch" :)